scrapy/http/request/__init__.py

```diff
@@ -103,3 +104,9 @@ def replace(self, *args, **kwargs):
             kwargs.setdefault(x, getattr(self, x))
         cls = kwargs.pop('cls', self.__class__)
         return cls(*args, **kwargs)
+
+    @classmethod
+    def from_curl(cls, curl_command):
```

Comment:
Implemented. I changed it to accept the same arguments as `Request`. Also documented that those arguments take preference over those contained in the "curl command". I was doubting if it's necessary to override them or if we could just extend (in case of `meta`, `headers` and `cookies` for example), but I decided just to override because if we allow to extend those fields and the programmer want to override them it could be counterintuitive. If someone want just to change one header it could be possible to create the Request and then change that specific header.

Tell me if you think I'm wrong or if you think it could be better to extend those arguments.